renovate.json

```diff
@@ -47,6 +47,9 @@
       "groupName": "IdentityModel.Tokens",
       "matchPackagePatterns": [
         "IdentityModel.Tokens"
+      ],
+      "allowedVersions": [
```

Comment:
`"allowedVersions"` does not appear to be an `array` but a `string`: https://docs.renovatebot.com/configuration-options/#allowedversions

Also could you alphabetize this?

Reply:
ah, missed that. I like to read the package name first and then the rest, but anyways